plugins/modules/vmware_guest_sendkey.py

```diff
@@ -247,6 +247,7 @@ def __init__(self, module):
             ('END', '0x4d', [('', [])]),
             ('CTRL_ALT_DEL', '0x4c', [('', ['CTRL', 'ALT'])]),
             ('CTRL_C', '0x06', [('', ['CTRL'])]),
+            ('CTRL_X', '0x1b', [('', ['CTRL'])]),
             ('RIGHTARROW', '0x4f', [('', [])]),
             ('LEFTARROW', '0x50', [('', [])]),
             ('DOWNARROW', '0x51', [('', [])]),
```

Comment:
Please add CTRL_X to the list of valid values in the description for keys_send.

Reply:
> Please add CTRL_X to the list of valid values in the description for keys_send.

@tejaswi-bachu Like [this](https://github.com/ansible-collections/community.vmware/pull/1376/commits/3aede43694f569944687fb88ce373008fb6a3266)?

Reply:
@mariolenz - Yes. Thanks.